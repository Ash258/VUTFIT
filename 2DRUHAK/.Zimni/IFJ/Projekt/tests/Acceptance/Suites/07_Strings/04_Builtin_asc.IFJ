' Asc builtin function

scope
	dim res as integer
	dim tmp as integer
	tmp = 0 - 10

	res = asc(!"test", 0)
	print res;
	res = asc(!"test", 1)
	print res;
	res = asc(!"\001", 1)
	print res;
	res = asc(!"\255", 1)
	print res;
	res = asc(!"\127", 1)
	print res;
	res = asc(!"\n", 1)
	print res;
	res = asc(!"asdasd", tmp)
	print res;
	res = asc(!"asdasd", 42)
	print res;
end scope
