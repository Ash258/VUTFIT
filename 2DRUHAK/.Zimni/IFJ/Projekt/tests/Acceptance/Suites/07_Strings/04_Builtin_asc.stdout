 0 116 1 255 127 10 0 0